packages/block-library/src/image/edit.js

```diff
@@ -879,6 +880,7 @@ export class ImageEdit extends Component {
 											left: showLeftHandle,
 										} }
 										onResizeStart={ () => {
+											this.props.showGrid();
```

Comment:
I think several blocks could make use of `ResizableBox` and the grid config together should we create a component that is already bound to reuse between the blocks that make use of it `GridResizableBox` or something?